contracts/src/Teleporter/ReceiptQueue.sol

```diff
@@ -8,90 +8,96 @@ pragma solidity 0.8.18;
 import "./ITeleporterMessenger.sol";
 
 /**
- * @dev ReceiptQueue is a convenience contract that creates a queue-like interface of
+ * @dev ReceiptQueue is a convenience library that creates a queue-like interface of
  * TeleporterMessageReceipt structs. It provides FIFO properties.
```

Comment:
can you add a note here about the need for functions to be `internal`

Reply:
added comments